examples/LGACSend.ino

```diff
@@ -0,0 +1,258 @@
+// Copyright 2017 xpokor22
+#include <IRremoteESP8266.h>
+#include <IRsend.h>
+#include <ir_LG.h>
+
+IRsend irsend(14);  // An IR LED is controlled by GPIO pin 14 (D5)
+
+// 0 : TOWER
+// 1 : WALL
+const unsigned int AC_TYPE  = 1;
+
+// 0 : cooling
+// 1 : heating
+unsigned int AC_HEAT = 1;
+
+// 0 : off
+// 1 : on
+unsigned int AC_POWER_ON = 0;
+
+// 0 : off
+// 1 : on --> power on
+unsigned int AC_AIR_ACLEAN = 0;
+
+// temperature : 18 ~ 30
+unsigned int AC_TEMPERATURE = 24;
+
+// 0 : low
+// 1 : mid
+// 2 : high
+// if AC_TYPE = 1, 3 : change
+unsigned int AC_FLOW = 0;
+
+const int AC_FLOW_TOWER[3] = {0, 4, 6};
+const int AC_FLOW_WALL[4] = {0, 2, 4, 5};
+
+unsigned long AC_CODE_TO_SEND;
+
+void ac_send_code(unsigned long code) {
```

Comment:
AcSendCode() .. as per https://google.github.io/styleguide/cppguide.html#Function_Names
Same for the other functions please.